proxy/src/main/java/com/wavefront/agent/ChannelStringHandler.java

```diff
@@ -46,6 +48,10 @@ public ChannelStringHandler(Decoder<String> decoder,
     this.pointHandler = pointhandler;
     this.preprocessor = preprocessor;
     this.metadataHandler = metadataHandler;
+
+    // check the property setting for logging raw data
+    String logRawDataProperty = System.getProperty("wavefront.proxy.lograwdata");
+    logRawDataFlag = logRawDataProperty != null && logRawDataProperty.equalsIgnoreCase("true");
```

Comment:
See vasily's commit, use a double instead from 0 to 1 to and a rate limiter to have control over how much to log

Reply:
got it, working